src/marqo/vespa/vespa_client.py

```diff
@@ -193,6 +198,8 @@ def wait_for_application_convergence(self, timeout: int = 120) -> None:
             except (httpx.TimeoutException, httpcore.TimeoutException):
                 logger.error("Marqo timed out waiting for Vespa application to converge. Will retry.")
 
+        logger.debug(f"Vespa application did not converge within {timeout} seconds, the convergence status is {self._get_convergence_status()}")
```

Comment:
Why don't you add the status to the error message instead? This debug message seems redundant

Reply:
I've moved it to error message